File: 프로그래머스/unrated/181913. 문자열 여러 번 뒤집기/문자열 여러 번 뒤집기.js
function solution(my_string, queries) {
    my_string = my_string.split('');
    
    queries.forEach(query => {
        const temp = my_string.slice(query[0], query[1] + 1);
        let j = 0;
        
        for (let i = query[1]; i >= query[0]; i--) {
            my_string[i] = temp[0 + j];
            j++;
        }
    })
    
    return my_string.join('');
}

File: 프로그래머스/unrated/181913. 문자열 여러 번 뒤집기/문자열 여러 번 뒤집기.java
class Solution {
    public String solution(String my_string, int[][] queries) {
        String[] split = my_string.split("");
        
        for (int[] query : queries) {
            int s = query[0], e = query[1], j = 0;
            for (int i = s; i <= (s + e) / 2; i++) {
                String temp = split[i];
                split[i] = split[e - j];
                split[e - j] = temp;
                j++;
            }
        }
        
        return String.join("", split);
    }
}